chemprop/nn/loss.py

```diff
@@ -27,15 +27,19 @@
 
 
 class LossFunction(nn.Module):
-    def __init__(self, task_weights: Tensor):
+    def __init__(self, task_weights: Tensor | None = None):
```

Comment:
does this need to be `Tensor | None`? what about `task_weights: ArrayLike = 1.0`? This corresponds to a default of equal-weight tasks as scalars are fully broadcasted.

Reply:
I thought about it more and agree 1.0 is a better default than None.